js-pkg/create-y-sweet-app/src/templates/nextjs/components/MyCollaborativeApp.tsx

```diff
@@ -0,0 +1,276 @@
+"use client";
+
+import Image from "next/image";
+import {
+  useMap,
+  useYSweetDebugUrl,
+  usePresence,
+  usePresenceSetter,
+} from "@y-sweet/react";
+import { useEffect, useState } from "react";
+import { randomColor } from "@/lib/colors";
+
+export function MyCollaborativeApp() {
+  const debugUrl = useYSweetDebugUrl();
+
+  const todos = useMap<boolean>("todos");
+
+  const setPresence = usePresenceSetter();
+  const presence = usePresence();
+
+  const [self, setSelf] = useState(() => ({ color: randomColor() }));
+  useEffect(() => setPresence(self), [setPresence, self]);
+
+  const others = Array.from(presence.entries());
+
+  return (
+    <div
```

Comment:
might be helpful for legibility to break this into three componets - hero, presence, todos?